deploy/canary/Dockerfile

```diff
@@ -1,10 +0,0 @@
-FROM golang:latest
```

Comment:
Docker hub can do builds on any push to any branch/tag & tag the docker image appropriately.